src/java.base/share/classes/java/security/spec/PSSParameterSpec.java

```diff
@@ -102,9 +107,15 @@
 
     /**
      * The PSS parameter set with all default values
+     * @deprecated The default values defined in the ASN.1 encoding in
+     *         PKCS#1 may become obsolete as time progresses. Thus, it is
+     *         recommended to explicitly specify all desired parameter
+     *         values with
+     *         {@link #PSSParameterSpec(String, String, AlgorithmParameterSpec, int, int) PSSParameterSpec}.
      *
      * @since 1.5
      */
+    @Deprecated(since="19", forRemoval=true)
```

Comment:
Do we really want to remove it in the future? It's awkward but probably not so harmful. Removing it may unnecessarily break existing codes. Same with the constructor below.

Reply:
Hmm, I am under the impression that we tend to use forRemoval=true when deprecating stuff, so users may pay more attention to it. This does not mean that we MUST remove it? Also, marking it for removal would give users more time in case something came up and removal become necessary?

Reply:
You are right that there is no rule that an API deprecated for removal must be removed. OK, you decide what the best way is.

Reply:
It's true that the chance of removing it is not that high (at least based on current situation). So, I removed the forRemoval=true part as you suggested.